wormchain/go.mod

```diff
@@ -3,27 +3,26 @@ module github.com/wormhole-foundation/wormchain
 go 1.16
 
 require (
-	github.com/CosmWasm/wasmd v0.28.0
-	github.com/CosmWasm/wasmvm v1.0.0
-	github.com/cosmos/cosmos-sdk v0.45.9
+	github.com/CosmWasm/wasmd v0.30.0
+	github.com/CosmWasm/wasmvm v1.1.1
+	github.com/cosmos/cosmos-sdk v0.45.11
 	github.com/cosmos/ibc-go/v3 v3.3.0
```

Comment:
we're still importing the old v3, are we mixing things up?

Reply:
good catch, this dep was stale.